docs/querying/api.md

```diff
@@ -1022,3 +1022,205 @@ $ curl -XPOST http://localhost:9090/api/v1/admin/tsdb/clean_tombstones
 ```
 
 *New in v2.1 and supports PUT from v2.9*
+
+## LangServer
+
+The following endpoints provides the different feature handled by LSP (Language Server Protocol) in a stateless way.
```

Comment:
```suggestion
This API is designed to help Web UIs provide IDE like features when writing PromQL queries, without introducing the complexity of the stateful [Language Server Protocol (LSP)](https://microsoft.github.io/language-server-protocol/).

Internally it uses the [PromQL Language Server](https://github.com/prometheus-community/promql-langserver).

The returned values follow JSON schemes specified by the LSP.
To deal with these types on the client side it might be helpful to use the [`vscode-languageserver-types`](https://www.npmjs.com/package/vscode-languageserver-types)  NPM package which provides the respective types.
```

Reply:
done